modules/qna/src/views/full/Components/ContentAnswerForm/index.tsx

```diff
@@ -87,6 +87,16 @@ const ContentAnswerForm: FC<Props> = ({
     })
   }
 
+  const contentTypeWeight = {
+    builtin_text: 0,
+    builtin_image: 1,
+    builtin_card: 2,
+    builtin_carousel: 3,
+    'builtin_single-choice': 4
+  }
+
+  const sortContentTypes = (a, b) => contentTypeWeight[a.value] - contentTypeWeight[b.value]
```

Comment:
Can you add the order to the content type instead ? You know what I think about hardcoding stuff around... Makes it complicated to change stuff and remember where stuff is